src/Compilers/CSharp/Portable/Emitter/Model/NamedTypeSymbolAdapter.cs

```diff
@@ -1043,6 +1048,9 @@ internal virtual IEnumerable<PropertySymbol> GetPropertiesToEmit()
                 }
             }
         }
+
+        internal virtual string ExtensionName
```

Comment:
>internal virtual string ExtensionName [](http://example.com/codeflow?start=8&length=37)

Since usage of this method goes beyond this file, I think it should be declared abstract.